app/cli/src/main/scala/org/bitcoins/cli/ConsoleCli.scala

```diff
@@ -2011,6 +2014,8 @@ object ConsoleCli {
         sttp
           .post(uri"http://$host:${config.rpcPort}/")
           .contentType("application/json")
+          .auth
+          .basic("bitcoins", config.rpcPassword)
```

Comment:
why not let the rpc user be set too like most other applications?

Reply:
Because there's only one user. It doest's matter what's its name.

Reply:
> why not let the rpc user be set too like most other applications?

We are copying what eclair does for their backend. I'm not sure how we can correctly map users to access to backend services (for instance, does Alice have access to "bob-wallet") and vice versa